examples/s3_static_files/README.md

```diff
@@ -0,0 +1,63 @@
+# AWS S3 Static Files
+
+This example shows how you can configure Nautobot to host static files on AWS S3.
+
+## Installation
+
+Nautobot can use [`django-storages`](https://django-storages.readthedocs.io/en/stable/) to publish files to S3 -- see the [docs](https://nautobot.readthedocs.io/en/stable/installation/nautobot/#remote-file-storage) for more information.
+
+TLDR:
+
+```shell
+$ echo django-storages >> $NAUTOBOT_ROOT/local_requirements.txt
+$ pip3 install django-storages
+```
+
+## Configuration
+
+The [`django-storages`](https://django-storages.readthedocs.io/en/stable/) library is quite powerful, please refer to their documentation for a more detailed explanation of the individual settings or for more information.
+
+In `nautobot_config.py` define the following configuration:
+
+```python
+STORAGE_BACKEND = "storages.backends.s3boto3.S3Boto3Storage"
+
+STORAGE_CONFIG = {
+    "AWS_ACCESS_KEY_ID": "...",
+    "AWS_SECRET_ACCESS_KEY": "...",
+    "AWS_STORAGE_BUCKET_NAME": "my-bucket-name",
+    "AWS_S3_REGION_NAME": "us-west-1",
+    "AWS_DEFAULT_ACL": "public-read",
+    "AWS_QUERYSTRING_AUTH": False,
+    "AWS_LOCATION": "subfolder/name/"
+    "USE_TZ": True,
```

Comment:
Could we potentially just do:

```
    "USE_TZ": USE_TZ,
```

It looks like the nautobot_config.py imports in the settings. Not sure if the potential for someone to set the global `USE_TZ` to something different than the `USE_TZ` stored within `STORAGE_CONFIG`.

Reply:
I think I'd either rather set it explicitly to `True` here or wait until PR #673 is merged and just remove it here

Reply:
That works. I'm not sure if #673 will make it into 1.1.0 or if we're going to get rid of the monkey patching completely in a later release.